controllers/apps/components/consensus_set_utils.go

```diff
@@ -188,6 +191,7 @@ func updateConsensusSetRoleLabel(cli client.Client,
 	patch := client.MergeFrom(pod.DeepCopy())
 	pod.Labels[constant.RoleLabelKey] = role
 	pod.Labels[constant.ConsensusSetAccessModeLabelKey] = string(roleMap[role].accessMode)
+	pod.Annotations[constant.LastRoleChangedEventTimestampAnnotationKey] = event.FirstTimestamp.Time.Format(time.RFC3339)
```

Comment:
maybe `event.LastTimestamp` a better choice?
as similar event can be merged into exist one by kubelet, if that happened, event with latest `event.LastTimestamp` is the most recent one:
1. role changed to leader, event1 created with `event. FirstTimestamp` set to T1
2. role changed to follower, event2 created with `event.FirstTimestamp` set to T2(T2>T1)
3. role changed to leader again, event 1 updated as similar role message generated, but the `event. FirstTimestamp` is still T1

Reply:
> maybe `event.LastTimestamp` a better choice? as similar event can be merged into exist one by kubelet, if that happened, event with latest `event.LastTimestamp` is the most recent one:
> 
> 1. role changed to leader, event1 created with `event. FirstTimestamp` set to T1
> 2. role changed to follower, event2 created with `event.FirstTimestamp` set to T2(T2>T1)
> 3. role changed to leader again, event 1 updated as similar role message generated, but the `event. FirstTimestamp` is still T1

fix done